README.md

```diff
@@ -710,14 +710,26 @@ fm.stop # callbacks are fired
 
 ### 3.7 Logging transitions
 
-To help debug your state machine, **FiniteMachine** provides `:log_transitions` option.
+To help debug your state machine, **FiniteMachine** provides a `:log_transitions` option.
 
 ```ruby
 FiniteMachine.new(log_transitions: true) do
   ...
 end
 ```
 
+A FiniteMachine instance can optionally be given a name:
+
+```ruby
+fm = FiniteMachine.new(name: 'TrafficLights', log_transitions: true) do
```

Comment:
Either add `fm` to the prior example or remove it in this one to make the section examples consistent.

Reply:
Ah! Ok, will do.